File: app/src/main/java/com/example/livedata/Blog.java
package com.example.livedata;

import com.google.gson.annotations.SerializedName;

public class Blog {
    @SerializedName("title")
    private String mTitle;
    @SerializedName("description")
    private String mDescription;
    @SerializedName("imageHref")
    private String mImageHref;

    public String getmTitle() {
        return mTitle;
    }

    public void setmTitle(String mTitle) {
        this.mTitle = mTitle;
    }

    public String getmDescription() {
        return mDescription;
    }

    public void setmDescription(String mDescription) {
        this.mDescription = mDescription;
    }

    public String getmImageHref() {
        return mImageHref;
    }

    public void setmImageHref(String mImageHref) {
        this.mImageHref = mImageHref;
    }
}

File: app/src/main/java/com/example/livedata/data/Blog.kt
package com.example.livedata.data

import com.google.gson.annotations.SerializedName

// data class to get array elements
data class Blog(
    @SerializedName("title")
     var title: String? = null,

    @SerializedName("description")
     var description: String? = null,

    @SerializedName("imageHref")
     var imageHref: String? = null)